### PR TITLE
Update default.py

### PR DESCRIPTION
**Pull request by RAcharya1**
Einthusan had reverted to using einthusan.com instead of einthusan.tv.  Changed all the '.tv' back to '.com'

### CODE REVIEW

File: plugin.video.einthusan/default.py [hunk 1]
```diff
@@ -464,7 +464,7 @@ def preferred_server(lnk, mainurl):
 		vidpath = lnk.split('.tv/')[1]
```
Comment (mazhisai) [inline plugin.video.einthusan/default.py]: This needs to be updated to .com/. this was the reason the plugin was not working for me.

Comment (RAcharya1) [reply plugin.video.einthusan/default.py]: ok. thanks. will apply fix

Comment (RAcharya1) [reply plugin.video.einthusan/default.py]: fix made and commited

Comment (RAcharya1) [reply plugin.video.einthusan/default.py]: let me know of any other issues during testing.

Comment (mazhisai) [reply plugin.video.einthusan/default.py]: This was working fine until two days back. It broke again today. For starters I think they now do Geo detection and take you to nearest server. I am not being redirected to einthusan.ca. second they are using cdn now for hosting binaries. I spent some time trying to get it working but it was beyond my understanding on how this script works.

**Comment by RAcharya1** [comment]
I will take a look at it again.  Maybe I overlooked the geo-location part of the code.

**Comment by reasonsrepo** [comment]
Thanks a lot for making a pull request and fixing the bugs, I've looked over the codes and fixed most of the things.

**Pull request closed without merging**